### ISSUE TITLE
[Error] Patch League: Failed to find fopen call!

### ISSUE THREAD

**Issue by KikiNeosquall** [moonshadow565/lolcustomskin-tools#8]
Hello there !
After the patch 11.2, LCS Manager doesn't load my skins, and I got this : [Error] Patch League: Failed to find fopen call!
Before the patch, it worked fine.
I tried to launch as administrator, even in moving LCS in another folder, that doesn't work.
I hope my log could help ! :)
[Log.txt](https://github.com/moonshadow565/lolcustomskin-tools/files/5859077/Log.txt)

**Comment by Berbab** [comment]
yeah i have the same problem, idk what to do!!!

**Comment by KikiNeosquall** [comment]
Hello ! I finally found something, and LCS works
[15:09]
1) Start LCS
2) Launch practice tool, and get the error
3) Left the game, but don't close LoL
4) Stop and run your skins in LCS
5) Start one game 
6) Enjoy.

**Comment by Berbab** [comment]
yeah thanks, but i've just tryied to unistall and reinstall the game and now it works fine too XD

**Comment by moonshadow565** [comment]
No idea if i'll get motivation to fix this one but this is a workaround that _should_ work:
0. Make sure LCS is **NOT** running
1. Get into practice mode
2. Wait for loading screen to finish
3. Run LCS while still in game
4. Close game 
5. Any runs after that should work

This should be done once per patch.
